lib/cocoapods/target/pod_target.rb

```diff
@@ -75,7 +82,7 @@ def scoped
     #
     def label
       if scoped?
-        "#{target_definitions.first.label}-#{root_spec.name}"
+        "#{root_spec.name}-#{scope_suffix}"
```

Comment:
Increase the length and reduces the overview because it breaks the order with fully de-duplicated pods.